src/allencell_ml_segmenter/thresholding/thresholding_view.py

```diff
@@ -0,0 +1,343 @@
+from pathlib import Path
+from typing import Optional
+
+from napari.utils.notifications import show_info  # type: ignore
+
+from allencell_ml_segmenter.core.dialog_box import DialogBox
+from allencell_ml_segmenter.main.i_experiments_model import IExperimentsModel
+from allencell_ml_segmenter.main.i_viewer import IViewer
+from allencell_ml_segmenter._style import Style
+from allencell_ml_segmenter.core.view import View, MainWindow
+from allencell_ml_segmenter.main.main_model import MainModel
+from allencell_ml_segmenter.prediction.prediction_folder_progress_tracker import (
+    PredictionFolderProgressTracker,
+)
+from allencell_ml_segmenter.prediction.service import ModelFileService
+from allencell_ml_segmenter.thresholding.thresholding_model import (
+    ThresholdingModel,
+    AVAILABLE_AUTOTHRESHOLD_METHODS,
+    THRESHOLD_RANGE,
+)
+from allencell_ml_segmenter.utils.file_utils import FileUtils
+
+from allencell_ml_segmenter.widgets.label_with_hint_widget import LabelWithHint
+from allencell_ml_segmenter.core.file_input_widget import (
+    FileInputWidget,
+)
+from allencell_ml_segmenter.core.file_input_model import (
+    FileInputModel,
+    InputMode,
+)
+
+from qtpy.QtWidgets import (
+    QLabel,
+    QVBoxLayout,
+    QHBoxLayout,
+    QSizePolicy,
+    QComboBox,
+    QGroupBox,
+    QRadioButton,
+    QPushButton,
+    QSlider,
+    QSpinBox,
+)
+from qtpy.QtCore import Qt
+
+
+class ThresholdingView(View, MainWindow):
+    """
+    View for thresholding
+    """
+
+    def __init__(
+        self,
+        main_model: MainModel,
+        thresholding_model: ThresholdingModel,
+        file_input_model: FileInputModel,
+        experiments_model: IExperimentsModel,
+        viewer: IViewer,
+    ):
+        super().__init__()
+
+        self._main_model: MainModel = main_model
+        self._experiments_model: IExperimentsModel = experiments_model
+        self._viewer: IViewer = viewer
+        self._thresholding_model: ThresholdingModel = thresholding_model
+
+        # To manage input files:
+        self._file_input_model: FileInputModel = file_input_model
+        self._input_files_service: ModelFileService = ModelFileService(
+            self._file_input_model
+        )
+
+        layout: QVBoxLayout = QVBoxLayout()
+        layout.setContentsMargins(0, 0, 0, 0)
+        layout.setSpacing(20)
+        self.setLayout(layout)
+        layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+        self.setSizePolicy(
+            QSizePolicy.Policy.Preferred, QSizePolicy.Policy.Maximum
+        )
+
+        # title
+        self._title: QLabel = QLabel("THRESHOLD", self)
+        self._title.setObjectName("title")
+        layout.addWidget(self._title, alignment=Qt.AlignmentFlag.AlignHCenter)
+
+        # selecting input image
+        self._file_input_widget: FileInputWidget = FileInputWidget(
```

Comment:
Why is this part of the thresholding view?  A model is already selected in the experiment model, so we should limit the workflow to only thresholding that.  

This needs to use the guardrails that we have put in place (user experiments) or else there are too many edge cases to handle